viewer/models.py

```diff
@@ -1901,3 +1901,57 @@ def __repr__(self) -> str:
             self.product_uuid,
             self.unit,
         )
+
+
+class ResultValueDataType(models.Model):
+    data_type = models.TextField(primary_key=True)
+    db_type = models.TextField()
+
+
+class ResultValueModifier(models.Model):
+    modifier = models.TextField(primary_key=True)
+    django_operator = models.TextField()
+
+
+class ResultUpload(models.Model):
+    target = models.ForeignKey(Target, null=True, on_delete=models.CASCADE)
+    upload_file = models.FileField(upload_to='assay_data/')
+    upload_date = models.DateTimeField(
+        null=True,
+        blank=True,
+        default=timezone.now,
+    )
+    uploaded_by = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        default=settings.ANONYMOUS_USER,
+    )
+
+
+class Result(models.Model):
```

Comment:
@kaliif  To match up better with the serializers etc, a more descriptive model name would be something like AssayResult? I am aware this might make many downfield changes. At the least,  a short function comment would help future folk know what this "Result" model is all about.